src/engine/client/cl_keys.cpp

```diff
@@ -599,7 +292,7 @@ void Console_Key( int key )
 	}
 
 	// ctrl-L clears screen
-	if (key == 'l' && keys[ K_CTRL ].down) {
+	if (key == Key::FromCharacter('l') && keys[ K_CTRL ].down) {
```

Comment:
It could be a constexpr, although I'd have to rewrite it in a less readable way to make the function consist of a single return statement. I expect the compiler to optimize it out regardless (see https://godbolt.org/g/yp5sua - it turns into 'mov rax 463856467969').